docs/Get Started/Quickstart - Your First Models.ipynb

```diff
@@ -11,8 +11,8 @@
     }
    },
    "source": [
-    "# Build your first SynapseML models\n",
-    "This tutorial provides a brief introduction to SynapseML. In particular, we use SynapseML to create two different pipelines for sentiment analysis. The first pipeline combines a text featurization stage with LightGBM regression to predict ratings based on review text from a dataset containing book reviews from Amazon. The second pipeline shows how to use prebuilt models through the Azure AI Services to solve this problem without training data."
+    "# Build your first SynapseML model\n",
+    "This article introduces how to build your first machine learning model using SynapseML and demonstrates how SynapseML simplifies complex machine learning tasks. We use SynapseML to create a small ML training pipeline that includes a featurization stage and a LightGBM regression stage. The pipeline predicts ratings based on review text from a dataset of Amazon book reviews. Finally, we showcase how SynapseML simplifies the use of prebuilt models to solve ML problems.\n"
```

Comment:
Not sure this is that much of an improvement

Reply:
using the original version